buffer.py

```diff
@@ -48,11 +48,16 @@ def __init__(self, buffer_id, url, arguments):
 
         self.delete_temp_file = arguments == "temp_pdf_file"
 
-        self.synctex_page_num = None
-        if arguments.startswith("synctex_page_num"):
-            self.synctex_page_num = int(arguments.split("=")[1])
+        self.synctex_info = [None, None, None, None]
+        if arguments.startswith("synctex_info"):
+            synctex_info = arguments.split("=")[1].split(":")
+            page_num = int(synctex_info[0])
+            pos_x = float(synctex_info[1])
+            pos_y = float(synctex_info[2])
+            sync_time = time.time()
```

Comment:
sync_time should remove after switch to QTimer.singleShot(5000, self.reset_synctex_indicator)